README.md

```diff
@@ -11,56 +11,66 @@ developer guide](http://pravega.io).
 
+Please, find the related documentation and instructions [here](https://github.com/pravega/pravega-samples/tree/master/hadoop-connector-examples).
 
 ## Scenarios
 | Example Name   | Description  | Language |
 | ------------- |:-----| :-----|
-| `turbineheatsensor` | It emulates parallel sensors producing temperature values (writers) and parallel consumers performing real-time statistics (readers) via Pravega client. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/iot)
-| `turbineheatprocessor` | A Flink streaming application for processing temperature data from a Pravega stream produced by the `turbineheatsensor` app. The application computes a daily summary of the temperature range observed on that day by each sensor. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/iot), [Scala](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/scala/io/pravega/examples/flink/iot)
-| `anomaly-detection` | A Flink streaming application for detecting anomalous input patterns using a finite-state machine. | [Java](https://github.com/pravega/pravega-samples/tree/master/anomaly-detection)
+| [`turbineheatsensor`](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-sensor) | It emulates parallel sensors producing temperature values (writers) and parallel consumers performing real-time statistics (readers) via Pravega client. | [Java](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-sensor/src/main/java/io/pravega/turbineheatsensor)
+| [`turbineheatprocessor`](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-processor) | A Flink streaming application for processing temperature data from a Pravega stream produced by the `turbineheatsensor` app. The application computes a daily summary of the temperature range observed on that day by each sensor. | [Java](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-processor/src/main/java/io/pravega/turbineheatprocessor), [Scala](https://github.com/pravega/pravega-samples/tree/master/scenarios/turbine-heat-processor/src/main/scala/io/pravega/turbineheatprocessor)
+| [`anomaly-detection`](https://github.com/pravega/pravega-samples/tree/master/scenarios/anomaly-detection) | A Flink streaming application for detecting anomalous input patterns using a finite-state machine. | [Java](https://github.com/pravega/pravega-samples/tree/master/scenarios/anomaly-detection/src/main/java/io/pravega/anomalydetection)
 
 
 # Build Instructions
 
+
+The `pravega-samples` project is prepared for working with [release artifacts](https://github.com/pravega/pravega/releases) of Pravega components, 
```

Comment:
These instructions are bit confusing because they are not super specific about how to configure the versions and so on. I suggest we start with instructions for how to run out of the box, and have some additional discussion with concrete examples to configure the samples to use a different pravega version.

Reply:
I tried to clarify the build instructions by distinguishing the `out-of-the-box` from the `build-from-source` options. All the mentions to `gradle.properties` are in the latter option. I also added an example of how to change the `gradle.properties` file to configure source builds. Let me know if the instructions are easier to follow now.